packages/editor/src/package/index.ts

```diff
@@ -7,6 +7,8 @@ export { defaultPlugins } from './config'
 export type { BaseEditor } from '@editor/core'
 
 export { EditorPluginType } from '@editor/types/editor-plugin-type'
+export { TemplatePluginType } from '@editor/types/template-plugin-type'
+export { type EditorVariant } from '@editor/package/storage-format'
```

Comment:
How will this be used? Could you perhaps add an example to Readme?

Reply:
I updated the readme. While doing so, I realized we can delete a whole bunch of other package exports because they were only necessary because we had `customPlugins`. :partying_face: